apps/src/p5lab/AnimationTab/PiskelEditor.jsx

```diff
@@ -210,6 +214,7 @@ class PiskelEditor extends React.Component {
       <iframe
         ref={iframe => (this.iframe = iframe)}
         style={this.props.style}
+        id="piskelFrame"
```

Comment:
Why is this being added?

Reply:
I added so that I can make sure I am getting the correct frame when passing the localeCode in the iframe.contentWindow.locale

const iframe = document.getElementById('piskelFrame');

Reply:
I'm pretty sure instead of `document.getElementById('piskelFrame');` you can do `this.iframe`.

Line 215 sets up the reference to `this.iframe`.